docs/jwst/pipeline/calwebb_spec2.rst

```diff
@@ -96,6 +96,110 @@ performed on the original (unresampled) data. The :ref:`cube_build <cube_build_s
++---------------------------------------+------------+--------------+-----------------+--------------+
+| :ref:`extract_2d <extract_2d_step>`   |  MOS, FS   |   MOS, FS    |    MOS, FS      |       MOS    |
++---------------------------------------+------------+--------------+-----------------+--------------+
+| :ref:`srctype <srctype_step>`         |  NONE      |    NONE      |      NONE       |      NONE    |
++---------------------------------------+------------+--------------+-----------------+--------------+
+| :ref:`wavecorr <wavecorr_step>`       |   ALL      |     ALL      |       ALL       |       ALL    |
++---------------------------------------+------------+--------------+-----------------+--------------+
+| :ref:`flat_field <flatfield_step>`    |            |              |                 |      NONE    |
++---------------------------------------+------------+--------------+-----------------+--------------+
+|              - D-FLAT                 |   ALL      |     ALL      |       ALL       |              |
++---------------------------------------+------------+--------------+-----------------+--------------+
+|              - S-FLAT                 |   ALL      |    NONE      |       ALL       |              |
++---------------------------------------+------------+--------------+-----------------+--------------+
+|              - F-FLAT                 |  NONE      |    NONE      |      NONE       |              |
++---------------------------------------+------------+--------------+-----------------+--------------+
+| :ref:`pathloss <pathloss_step>`       |  NONE      |    NONE      |      NONE       |      NONE    |
++---------------------------------------+------------+--------------+-----------------+--------------+
+| :ref:`barshadow <barshadow_step>`     |  NONE      |    NONE      |      NONE       |      NONE    |
++---------------------------------------+------------+--------------+-----------------+--------------+
+| :ref:`photom <photom_step>`           |  NONE      |    NONE      |      NONE       |      NONE    |
++---------------------------------------+------------+--------------+-----------------+--------------+
+| :ref:`resample_spec <resample_step>`  |  MOS, FS   |    NONE      |    MOS, FS      |      NONE    |
++---------------------------------------+------------+--------------+-----------------+--------------+
+| :ref:`cube_build <cube_build_step>`   |   IFU      |    NONE      |       IFU       |      NONE    |
++---------------------------------------+------------+--------------+-----------------+--------------+
+| :ref:`extract_1d <extract_1d_step>`   |   ALL      |    NONE      |       ALL       |      NONE    |
++---------------------------------------+------------+--------------+-----------------+--------------+
+
+In the resample_spec and cube build steps, the spectra are transformed to a space of (wavelength,
```

Comment:
Could put back-tics around the resample_spec and cube_build step names. Or perhaps even make them links (references) to the steps.

Reply:
Good suggestion, fixed